core/src/main/java/hudson/model/StreamBuildListener.java

```diff
@@ -40,16 +41,26 @@
  * @author Kohsuke Kawaguchi
  */
 public class StreamBuildListener extends StreamTaskListener implements BuildListener {
+    private final Closeable[] additionalClosables;
```

Comment:
Making this field be non`transient` seems to cause [many legitimate-looking test failures](https://ci.jenkins.io/job/Core/job/jenkins/job/PR-2953/1/testReport/).